src/rp/util/logic.clj

```diff
@@ -0,0 +1,8 @@
+(ns rp.util.logic)
+
+(defn only-one
+  "Returns x in xs if x is the only truthy x, falsey otherwise."
```

Comment:
What do you think about replacing "falsey" with "nil" in both docstrings?

Other than that suggestion, this PR has my 👍